flights_nyc13/pre_processing.R

```diff
@@ -0,0 +1,33 @@
+###20.08
```

Comment:
please add all libraries 
install.packages("nycflights13")
library(nycflights13)
library(dplyr)
library(tidyverse)
library(RColorBrewer)
library(gplots)

Reply:
add initializing of datasets:
flights <- flights
airlines <- airlines
airports <- airports
planes <- planes
weather <- weather

Reply:
/